src/Microdown-HTMLExporter-Tests/MicHTMLVisitorTest.class.st

```diff
@@ -147,25 +147,30 @@ MicHTMLVisitorTest >> testCreateAnchorWithLink [
 MicHTMLVisitorTest >> testCreateAnnotationCompound [
 
 	| result |
-	
 	result := writer convertMicString: '# This is a title
-!!LabelOfAnnotatedBlock Body of annotated block
+!!note Body of annotated block
 
 this is another string'.
-	self assert: result trimBoth equals: '<h1>This is a title</h1>', newLine ,
-'<section id="LabelOfAnnotatedBlock"><label for="LabelOfAnnotatedBlock">Body of annotated block</label></section>', newLine ,
-'<p>this is another string</p>'.
+	self
```

Comment:
To me it looks strange to create a heading 1 for a note. 
Why did you suggest this?